test/e2e/downwardapi_volume.go

```diff
@@ -18,57 +18,179 @@ package e2e
 
 import (
 	"fmt"
+	"strings"
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/util"
 
 	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 )
 
+var timeOut = "15s"
```

Comment:
similarly, see if one of the predefined timeout values in util is sufficient. if not, at least choose a more descriptive name, maybe?